trunk/NDHMS/Routing/Reservoirs/reservoir_tests.F

```diff
@@ -5,9 +5,24 @@ program reservoir_unit_tests
     implicit none
```

Comment:
How is this built?

Reply:
I see it in your comments on this PR on how to make and run. Can you please add those instructions to the top of this file?

It looks like this file was there previously, so this should probably have been done previously....

Reply:
Type 'make test' in the Reservoir directory. Then run with './reservoir_tests'.

Reply:
Functionally, this PR is not that closely tied to the code, it's at least 1 step away. Putting the instructions at the top of the file will help close that gap.

Reply:
Okay will do.